src/index.ts

```diff
@@ -42,18 +42,12 @@ export default function cssInjectedByJsPlugin({
             enforce: 'post',
             name: 'vite-plugin-css-injected-by-js',
             config(config, env) {
-                if (env.command === 'build') {
-                    if (!config.build) {
-                        config.build = {};
-                    }
-
-                    if (relativeCSSInjection == true) {
-                        if (config.build.cssCodeSplit == false) {
```

Comment:
To fix what the title says it's enough to change this to: `if (!config.build.cssCodeSplit) {`